code/sound/voicerec.cpp

```diff
@@ -41,6 +41,10 @@
 #include "osapi/osapi.h"
 #include "playerman/player.h"
 #include "ship/ship.h"
+#include "cfile/cfile.h"
+#include "osapi/osapi.h"
+
+#include <SDL_syswm.h>
```

Comment:
Duplicate include of <SDL_syswm.h>

Also, these changes here (code/sound/voicerec.cpp:44) don't appear to be relevant to the PR.